include/sys/util.h

```diff
@@ -544,6 +544,16 @@ uint8_t u8_to_dec(char *buf, uint8_t buflen, uint8_t value);
  */
 #define EMPTY
 
+/**
+ * @brief Macro that expands to its argument
+ *
+ * This is useful in macros like @c FOR_EACH() when there is no
+ * transformation required on the list elements.
+ *
+ * @param V any value
+ */
+#define IDENTITY(V) V
```

Comment:
Part of me wonders if this should be renamed UTIL_IDENTITY or so. But we have pretty questionable namespacing in this file already, so it must not be a real issue.